guides/source/active_record_validations.md

```diff
@@ -365,6 +365,36 @@ end
 
 The default error message for this helper is _"doesn't match confirmation"_.
 
+### `comparison`
+
+This check will validate a comparison between any two comparable values.
+If an option is not passed, all validations pass. It accepts a value, proc, or symbol.
+Any class that includes Comparable can be compared.
+
+```ruby
+class Promotion < ApplicationRecord
+  validates :start_date, compare: { greater_than: :end_date }
```

Comment:
I think it needs to be `validates :start_date, comparison: { greater_than: :end_date }` given the name of the validator, doesn't it?

Reply:
Yes! Sorry about that. Compare was the old name.